Can't compile on F39

I compile Cemu roughly once a month using my project which compiles Cemu in a F39 container
https://github.com/leifliddy/podman-cemu-builder

It's really easy to repro as you only need to run a couple of commands.....
Something has changed over the last week or two.

During the build phase I'm seeing this error 
```
+ cmake -S . -B build -DCMAKE_BUILD_TYPE=release -DCMAKE_C_COMPILER=/usr/bin/gcc -DCMAKE_CXX_COMPILER=/usr/bin/g++ -G Ninja
-- Bootstrapping vcpkg before install
-- Bootstrapping vcpkg before install - done
-- Running vcpkg install
error: failed to execute: /usr/bin/git --git-dir=/root/cemu/dependencies/vcpkg/.git --work-tree=/root/cemu/dependencies/vcpkg/buildtrees/versioning_/versions/boost-modular-build-helper/843c4331dc1453c80f99e25be4513c99074c5cf2_1293.tmp -c core.autocrlf=false read-tree -m -u 843c4331dc1453c80f99e25be4513c99074c5cf2
vcpkg was cloned as a shallow repository in: /root/cemu/dependencies/vcpkg/.git
Try again with a full vcpkg clone.
error: git failed with exit code: (128).
fatal: failed to unpack tree object 843c4331dc1453c80f99e25be4513c99074c5cf2
note: while checking out port boost-modular-build-helper with git tree 843c4331dc1453c80f99e25be4513c99074c5cf2
-- Running vcpkg install - failed
CMake Error at dependencies/vcpkg/scripts/buildsystems/vcpkg.cmake:899 (message):
  vcpkg install failed.  See logs for more information:
  /root/cemu/build/vcpkg-manifest-install.log
Call Stack (most recent call first):
  /usr/share/cmake/Modules/CMakeDetermineSystem.cmake:148 (include)
  CMakeLists.txt:37 (project)
```

There's something off with that command -- if I try and run it manually I get this 
```
[root@cemu_builder ~]# /usr/bin/git --git-dir=/root/cemu/dependencies/vcpkg/.git --work-tree=/root/cemu/dependencies/vcpkg/buildtrees/versioning_/versions/boost-modular-build-helper/843c4331dc1453c80f99e25be4513c99074c5cf2_1293.tmp -c core.autocrlf=false read-tree -m -u 843c4331dc1453c80f99e25be4513c99074c5cf2
fatal: failed to unpack tree object 843c4331dc1453c80f99e25be4513c99074c5cf2
```

Any help would be appreciated.

damn, now it's failing at 
```
error: building libxcrypt:x64-linux failed with: BUILD_FAILED
Elapsed time to handle libxcrypt:x64-linux: 5.9 s
Please ensure you're using the latest port files with `git pull` and `vcpkg update`.
Then check for known issues at:
  https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+libxcrypt
You can submit a new issue at:
  https://github.com/microsoft/vcpkg/issues/new?title=[libxcrypt]+Build+error+on+x64-linux&body=Copy+issue+body+from+%2Froot%2Fcemu%2Fbuild%2Fvcpkg_installed%2Fvcpkg%2Fissue_body.md

-- Running vcpkg install - failed
CMake Error at dependencies/vcpkg/scripts/buildsystems/vcpkg.cmake:899 (message):
  vcpkg install failed.  See logs for more information:
  /root/cemu/build/vcpkg-manifest-install.log
Call Stack (most recent call first):
  /usr/share/cmake/Modules/CMakeDetermineSystem.cmake:148 (include)
  CMakeLists.txt:37 (project)
```

hmmmmmm
````
configure:5212: result: no
configure:5212: checking for endian.h
configure:5212: /usr/bin/cc -c -fPIC -g  conftest.c >&5
configure:5212: $? = 0
configure:5212: result: yes
configure:5212: checking for sys/endian.h
configure:5212: /usr/bin/cc -c -fPIC -g  conftest.c >&5
conftest.c:56:10: fatal error: sys/endian.h: No such file or directory
   56 | #include <sys/endian.h>
      |          ^~~~~~~~~~~~~~
compilation terminated.
configure:5212: $? = 1
configure: failed program was:
| /* confdefs.h */
| #define PACKAGE_NAME "xcrypt"
| #define PACKAGE_TARNAME "libxcrypt"
| #define PACKAGE_VERSION "4.4.36"
| #define PACKAGE_STRING "xcrypt 4.4.36"
| #define PACKAGE_BUGREPORT "https://github.com/besser82/libxcrypt/issues"
| #define PACKAGE_URL "https://github.com/besser82/libxcrypt"
| #define PACKAGE "libxcrypt"
| #define VERSION "4.4.36"
```

--- a/Containerfile
+++ b/Containerfile
@@ -4,7 +4,8 @@ COPY files/bashrc /root/.bashrc
 COPY files/bashrc-default /root/.bashrc.d/default
 
 RUN dnf update -y &&\
-    dnf install -y bison clang cmake cubeb-devel findutils freeglut-devel git glm-devel gtk3-devel kernel-headers libgcrypt-devel libsecret-devel libtool libusb1-devel nasm ninja-build perl-FindBin perl-File-Compare perl-File-Copy perl-IPC-Cmd rsync systemd-devel vcpkg vim-enhanced wayland-protocols-devel wget zip zlib-devel &&\
+    dnf install -y bison clang cmake cubeb-devel findutils freeglut-devel git glm-devel gtk3-devel kernel-headers libgcrypt-devel libsecret-devel libtool libusb1-devel nasm ninja-build python3-pip perl-open perl-FindBin perl-File-Compare perl-File-Copy perl-IPC-Cmd rsync systemd-devel vcpkg vim-enhanced wayland-protocols-devel wget zip zlib-devel &&\
+    pip3 install jinja2 passlib &&\
     find /root/ -type f | grep -E 'anaconda-ks.cfg|anaconda-post-nochroot.log|anaconda-post.log|original-ks.cfg' | xargs rm -f &&\
     echo 'defaultyes=True' >> /etc/dnf/dnf.conf
 

--- a/build.scripts/01-build.cemu.sh
+++ b/build.scripts/01-build.cemu.sh
@@ -3,6 +3,7 @@
 set -e
 
 cemu_dir='/root/cemu'
+vcpkg_dir="$cemu_dir/dependencies/vcpkg"
 cemu_git_url='https://github.com/cemu-project/Cemu'
 
 while getopts :r arg
@@ -17,9 +18,10 @@ done
 
 [[ $rebuild = true ]] && git clone --recursive $cemu_git_url $cemu_dir
 
-# perform git pull
-pushd $cemu_dir
-git pull --recurse-submodules
+# perform vcpkg update
+pushd $vcpkg_dir
+git fetch --unshallow
+git pull origin master
 popd
 
 # remove build directory

--- a/build.scripts/01-build.cemu.sh
+++ b/build.scripts/01-build.cemu.sh
@@ -20,7 +20,7 @@ done
 
 # perform vcpkg update
 pushd $vcpkg_dir
-git fetch --unshallow
+git fetch --unshallow || true
 git pull origin master
 popd
 

